batches_test.go

```diff
@@ -0,0 +1,67 @@
+package packngo
+
+import (
+	"fmt"
+	"testing"
+)
+
+var batchID string
+
```

Comment:
All the acceptance tests **must** clean up created resources. We really need to figure out how the batches are destroyed, because without this, I can't imagine how the fuctionality would be useful in Terraform or Juju.

Maybe try to issue a HTTP DELETE to the batch URI path. We really need input from Packet devs on this.

Reply:
It is possible to delete devices created by batch POST, but that doesn't remove the batch itself.